spyderlib/plugins/shortcuts.py

```diff
@@ -176,6 +138,7 @@ def __init__(self, parent, context, name, sequence, shortcuts):
         self.text_new_sequence = CustomLineEdit(self)
         self.text_new_sequence.setPlaceholderText(sequence)
         self.helper_button = HelperToolButton()
+        self.label_warning = QLabel('<br><br>')

```

Comment:
@goanpeca, please move forward with this (and my other suggestions) to have this ready for beta1 :-)